_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -138,6 +141,7 @@ $palettes: (
 // using black-t because using a gray looks weird when it appears over black
 $shadow: transparentize(palette(grayscale, black-t), .5);
 
+
```

Comment:
Yup, two spaces in between sections of a Sass file, like:

``scss
// ----------------------------
// #UNITS
// ----------------------------
$em-base: 16; // deliberately sets bourbon-based em-base (http://bourbon.io/docs/#em-base)


// ----------------------------
// #GRID
// ----------------------------
// grid - baseline
$baseline:                   20 !default;
$baseline-vertical:          ($baseline*2) !default;
$baseline-horizontal:        $baseline !default;

// grid - dimensions
$grid-max-width:             rem(1170) !default;
$grid-column-count:          12 !default;
$grid-gutter-width:          (1/3) !default;  // set to fraction to be proportional to fluid column widths (see - http://susydocs.oddbird.net/en/latest/settings/#gutters)
$grid-container-position:    center !default;
$grid-gutter-position:       after !default;
$grid-manual-gutter-position: inside !default;
$grid-column-width:          false !default;  // set to false for fluid columns by default (see - http://susydocs.oddbird.net/en/latest/settings/#column-width)
$grid-debug-color:           rgba(203, 89, 141, 0.33) !default;

// grid - breakpoints
$bp-screen-sm:                320px !default;
$bp-screen-md:                768px !default;
$bp-screen-lg:                1024px !default;
$bp-screen-xl:               1280px !default;


// ----------------------------
// #COLORS
// ----------------------------
// color: config
$transparent:               rgba(0, 0, 0, 0);

$gray:                      rgb(167, 164, 164);
$gray-cool:                 rgb(158, 177, 185);
$red:                       rgb(178, 6, 16);
$yellow:                    rgb(253, 188, 86);
$green:                     rgb(37, 184, 90);
$blue:                      rgb(0, 121, 188);
$pink:                      rgb(203, 89, 141);

// colors: brand
$brand-primary:             $blue !default;
$brand-primary-accent:      rgb(14, 166, 236) !default;
$brand-secondary:           $pink !default;
$brand-secondary-accent:    rgb(242, 108, 170) !default;

// colors: utilities
$success: $green !default;
$warning: $yellow !default;
$error: $red !default;
```